src/components/uploader/OperationUploader.tsx

```diff
@@ -19,26 +19,33 @@ import { wrapErrorMessage } from 'utils/wrapErrorMessage'
 
 const operationPatch = (operation: object): object => {
   // this part is quite dirty, do not use in other parts
-
   // backend compatibility of minimum_required
-  if (!operation['minimum_required'] || operation['minimum_required'] == 'v4.0') {
+  if (
+    !operation['minimum_required'] ||
+    operation['minimum_required'] == 'v4.0'
```

Comment:
Just noticed this: please always use tripe-equality test (`===`), avoid implicit type cast, and when implicit type cast is needed, use explicit type cast.

Reply:
Great! 
I'm curious why [eslint eqeqeq](https://eslint.org/docs/latest/rules/eqeqeq) is not enabled.